app/views/admin/item_holds/index.html.erb

```diff
@@ -1,4 +1,12 @@
 <%= render "admin/items/profile" do %>
+<div class="btn-group mt-2">
+  <%= link_to_unless params[:inactive].blank?, "Active", url_for, method: :get, class: "btn btn-sm" do %>
+    <button class="btn btn-sm active">Active</button>
+  <% end %>
+  <%= link_to_unless_current "Inactive", url_for(inactive: true), method: :get, class: "btn btn-sm" do %>
```

Comment:
And the text of this link can be "Ended". I think that will be more familiar to users.